tensorboard/webapp/metrics/views/right_pane/scalar_column_editor/scalar_column_editor_component.scss

```diff
@@ -46,3 +56,27 @@ $_accent: map-get(mat.get-color-config($tb-theme), accent);
 .highlight-top {
   border-top: 2px solid mat.get-color-from-palette($_accent);
 }
+
+.footer {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  width: 100%;
+  height: 39px;
```

Comment:
rather than specifying height I think you should just let it naturally size based on its contents and some padding. I would:
- remove height.
- add `padding: 4px`;

That has the additional benefit of having some padding between the 'close' button and the elements and things surrounding it (the border at the top and right, and the bottom of the window), which you typically want to do. 4px looks good but you could also experiment with other low even numbers like 6px or 8px.

Reply:
Works for me.